### PR TITLE
Feature/email submit 1

### PR DESCRIPTION
**Pull request by zchenbonterra**

### CODE REVIEW

**Comment by zchenbonterra** [comment]
I am able to get email submission from users on slack, but I am getting the following error.

Error: {error: SyntaxError: Unexpected token o in JSON at position 0 at Object.parse (<anonymous>) at XMLHt…, text: "ok"}

File: src/app/app.component.ts [hunk 1]
```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'app';
+
```
Comment (GiorgioNatili) [inline src/app/app.component.ts:9]: Don't remove the title, let's call it `SwiftFest Slack Channel`

File: src/app/app.module.ts [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-    AppComponent
+    AppComponent,
+    LandingPageComponent
   ],
   imports: [
-    BrowserModule
+    BrowserModule,
+    FormsModule,
+    HttpClientModule,
+
```
Comment (GiorgioNatili) [inline src/app/app.module.ts:18]: extra empty line

File: src/app/landing-page/landing-page.component.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+<p>{{emailInput}}</p>
+<input type="text" [(ngModel)]="emailInput">
```
Comment (GiorgioNatili) [inline src/app/landing-page/landing-page.component.html:3]: I don't like too much bi-directional data binding, why are you using it?

File: src/app/landing-page/landing-page.component.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+<button (click)="submitEmail(emailInput)">
+  submit email
```
Comment (GiorgioNatili) [inline src/app/landing-page/landing-page.component.html:5]: In a separate PR please integrate https://github.com/ngx-translate/core to store labels outside of the code base.

File: src/app/landing-page/landing-page.component.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  selector: 'app-landing-page',
+  templateUrl: './landing-page.component.html',
+  styleUrls: ['./landing-page.component.scss']
+})
+export class LandingPageComponent implements OnInit {
+  emailInput: string;
+
+  constructor(private http: HttpClient) { }
+
+  ngOnInit() {
```
Comment (GiorgioNatili) [inline src/app/landing-page/landing-page.component.ts:16]: Remove it if not needed

**Review by GiorgioNatili — CHANGES_REQUESTED** [review]

**Pull request merged**